### PR TITLE
Libanos-related work

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: 1001-2000/LIT1473GadlaL.xml [hunk 1]
```diff
@@ -84,7 +87,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             </listBibl>
          </div>
-         <div type="edition">
+         <div type="edition" xml:id="recension2">
+            <note>Recension 2 provided after <bibl>
+               <ptr target="bm:Bausi2003Libanos"/>
+            </bibl></note>
             <div type="textpart" subtype="incipit" xml:lang="gez">
                <ab>
                 ስምዑ፡ እንግርክሙ፡ ዘንተ፡ ነገረ፡ በእንተ፡ <gap reason="omitted"/> ዘከመ፡ ሕይወ፡ ውስተ፡ ምድር፡ ፭፻፷ወ፯፡ ዓመተ፡ <gap reason="omitted"/>
```
Comment (DenisNosnitsin1970) [inline 1001-2000/LIT1473GadlaL.xml:96]: Please check the word ሕይወ፡

File: 1001-2000/LIT1473GadlaL.xml [hunk 2]
```diff
@@ -84,7 +87,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                </bibl>
             </listBibl>
          </div>
-         <div type="edition">
+         <div type="edition" xml:id="recension2">
+            <note>Recension 2 provided after <bibl>
+               <ptr target="bm:Bausi2003Libanos"/>
+            </bibl></note>
             <div type="textpart" subtype="incipit" xml:lang="gez">
```
Comment (DenisNosnitsin1970) [inline 1001-2000/LIT1473GadlaL.xml:94]: I wonder if this is the real incipit. I see that the quote is found on p. 8; but before there seems to be some text, I wonder if §1 and §2 (p. 1-2 of the text edition) could provide incipit.

File: 1001-2000/LIT1473GadlaL.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1473GadlaL.xml:96]: It should be ሐይወ፡

File: 1001-2000/LIT1473GadlaL.xml [hunk 3]
```diff
@@ -52,6 +52,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+         <abstract><p>The Gadl of St Libānos (Maṭāʿ) exists in three different recensions. 
+            The text is attributed to ʾElyas, the Bishop of Aksum (cp. <ref type="mss" corresp="ESmgm012"></ref>, <locus target="#17ra">17ra</locus>).</p></abstract>
```
Comment (DenisNosnitsin1970) [inline 1001-2000/LIT1473GadlaL.xml:56]: ʾElyās

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: 1001-2000/LIT1473GadlaL.xml (hunk 1 above)
Comment (eu-genia) [reply 1001-2000/LIT1473GadlaL.xml:96]: This incipit was not inserted by me as you can see from the file, this is an old record by MV, I don't have Bausi's edition at hand now - can you check please?

Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1473GadlaL.xml:150]: Should I type it in?

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: 1001-2000/LIT1473GadlaL.xml (hunk 2 above)
Comment (eu-genia) [reply 1001-2000/LIT1473GadlaL.xml:148]: sure, if you feel like typing :) i will then wait to merge

**Pull request merged**